Instruments/bridge12.py

```diff
@@ -404,25 +380,46 @@ def grab_consist_power():
         self.tuning_curve_data[sweep_name+'_freq'] = freq
         self.last_sweep_name = sweep_name
         return rxvalues, txvalues
+    def handle_midpoint_failure(self):
+        result = input("Couldn't fine the midpoint; maybe the wg didn't turn on completely, try again?")
+        if result.lower().startswith("y"):
+            wg_engaged = False
+        else:
+            self.set_rf(False)
+            self.set_wg(False)
+            raise ValueError("The reflection of the first point is the same or lower than the rx of the dip, which doesn't make sense -- check %gdBm_%s"%(10.0,'rx'))
```

Comment:
handles error that is run into with dip locking - previously added in HP branch when B12 was out but figured it's good to keep here

Reply:
okay got rid of function and instead put in except block with better explanation in the input statemtne